locales/it/index.yml

```diff
@@ -3333,7 +3333,7 @@ features:
         bottomSheet:
           about:
             title: "Chi è?"
-            subtitle: "È l'ente riconosciuto dallo Stato a fornirti la versione digitale dei tuoi documenti."
+            subtitle: "È l’ente riconosciuto dallo Stato a fornirti la versione digitale dei tuoi documenti.\n\n Per sapere come tratta i tuoi dati consulta l’[Informativa Privacy](https://io.italia.it/informativa-ipzs). "
```

Comment:
Can we use the URL stored in the .env file and use i18n concatenation to handle it? Does it make sense?

Reply:
Thanks for the suggestion, fixed!